tools/setup.py

```diff
@@ -27,7 +27,7 @@
     'nose>=1.3.7',
     'readme_renderer>=17.2',
     'requests',
-    'pyyaml~=5.2',
+    'pyyaml==5.3.1',
```

Comment:
PyYAML 5.3.1 has CVE that will trigger security alerts: https://github.com/yaml/pyyaml/issues/724#issuecomment-1638909203

Reply:
A security has already been triggered: https://github.com/Azure/azure-cli/security/dependabot/29